docs/conf.py

```diff
@@ -34,29 +34,28 @@
     "aliases": {
         "title": "Aliases",
         "description": (
-            "PlasmaPy provides short-named (alias) versions of the most "
-            "common plasma functionality.  These aliases are only given to "
-            "functionality where there is a common lexicon in the community, "
-            "for example `~plasmapy.formulary.frequencies.plasma_frequency` "
-            " has the alias `~plasmapy.formulary.frequencies.wp_`.  All aliases "
-            "in PlasmaPy are denoted with a trailing underscore ``_``."
+            """
+            PlasmaPy provides :term:`aliases` of the most common plasma
+            functionality for user convenience. Aliases in PlasmaPy are
+            denoted with a trailing underscore.
+            """
         ),
         "dunder": "__aliases__",
     },
     "lite-functions": {
         "title": "Lite-Functions",
         "description": (
             """
-            Much of PlasmaPy's functionality incorporates `Astropy units
-            <https://docs.astropy.org/en/stable/units/>`_ for user convenience and
-            to mitigate calculation errors from inappropriate units, but this
-            comes at the sacrifice of speed.  While this penalty is not significant
-            for typical use, it can become substantial during intensive numerical
-            calculations. **Lite-functions** are introduced for the specific case
-            where speed matters, but **[USER NOTICE]** this comes with the
-            reduction of safeguards so a user needs to know what they are doing!
-            For additional details look to the glossary entry for
-            :term:`lite-function`.
+            :term:`Lite-functions` are lightweight versions of existing
```

Comment:
```suggestion
            :term:`Lite-functions` are lite-weight versions of existing
```

[optional] Just to stay consistent with the lexicon.